app/controllers/auth_tokens_controller.rb

```diff
@@ -21,7 +21,7 @@ def index
   end
 
   def show
-    authenticate!
+    authenticate!(:token)
```

Comment:
It looks like you're passing an arg here but in the warden file, it doesn't take an arg